lib/blocks.php

```diff
@@ -71,6 +71,7 @@ function gutenberg_reregister_core_block_types() {
 				'post-author.php'               => 'core/post-author',
 				'post-comment.php'              => 'core/post-comment',
 				'post-comment-author.php'       => 'core/post-comment-author',
+				'comment-avatar.php'            => 'core/comment-avatar',
```

Comment:
It would be nice to keep names aligned:
```suggestion
				'post-comment-avatar.php'            => 'core/post-comment-avatar',
```

What's the relationship between Post Comment Author and Post Comment Avatar? Can you extend the PR's description to better explain the goals here?

Reply:
> Keep names aligned: 

There will be an imminent refactor, removing the post prefix in all comment blocks. [See comment](https://github.com/WordPress/gutenberg/issues/34994#issuecomment-929291106)

> Relationship between Post Comment Author and Post Comment Avatar

Avatar is just the image of the author of the comment. It can be edited in size, padding and border-radius. The author would be a combination of Avatar, Name and maybe a little description. Of course, I can extend the PR's description. Will do it!

Reply:
> There will be an imminent refactor, removing the post prefix in all comment blocks. See comment

I wasn't aware of that. I guess we will have to deprecate all existing blocks then. They are experimental and only available in the Gutenberg plugin so it shouldn't be that big of a deal.

Reply:
Reading again the comment from @mtias:

> As we work through these, I'd like to drop all the "Post" prefixes and just settle on "Comment Content", "Comment Author", etc.

Maybe we should update `title` and `description` as the first step, but leave folder names, file names, and block names prefixed with `post-` so we don't have to worry about backward compatibility for now.

Reply:
Agree with that. Done!

Reply:
I think you missed changes in this file, the rest looks great 😄